frontend/src/routes.js

```diff
@@ -19,29 +17,31 @@ const api_url = process.env.API_URL;
 
 // Add routes
 app.get('/', async (req, res) => {
-    const response = await fetch(api_url+'api/',{method:'GET',headers:{}})
+    const response = await fetch(api_url+'/api/',{method:'GET',headers:{}})
     const data = await response.json();
     res.render('homePage', {test: data}); 
 
   });
 
 app.get('/job-roles', async (req, res) => {
-    const response = await fetch(api_url+'job-roles',{method:'GET',headers:{}})
+    const response = await fetch(api_url+'/job-roles',{method:'GET',headers:{}})
     const roleData = await response.json();
     res.render('jobRolesPage', {items: roleData}); 
-
 });
 
 app.get('/job-roles/:id', async (req, res) => {
     var id = req.params.id;
-    console.log(api_url+'job-roles/'+id)
-    const response = await fetch(api_url+'job-roles/'+id,{method:'GET',headers:{}})
+    console.log(api_url+'/job-roles/'+id)
+    const response = await fetch(api_url+'/job-roles/'+id,{method:'GET',headers:{}})
     const roleData = await response.json();
-    const bandResponse = await fetch(api_url+'bands/'+roleData.bandID,{method:'GET',headers:{}})
+    const specResponse = await fetch(api_url+'/job-roles/job-spec/'+id,{method:'GET',headers:{}})
```

Comment:
this url call won't be necessary when you change the backend to provide information to job-roles/{id}